opentracing_instrumentation/client_hooks/__init__.py

```diff
@@ -60,7 +64,7 @@ def install_patches(patchers='all'):
     if patchers is None or patchers == 'all':
         install_all_patches()
         return
-    if type(patchers) is list:
+    if _valid_args(patchers):
```

Comment:
while you're here, do you mind reformatting this function to make it look like what you've for install_client_interceptors? ie reduce the level of indentation

Reply:
of course -- doing that :)